client-app/src/pages/account/store/modules/orders-list/actions.ts

```diff
@@ -15,6 +15,8 @@ export const actions: ActionTree<OrdersListState, RootState> = {
     searchCriteria.pageNumber = context.state.ordersList.listConfig.pageNumber;
     searchCriteria.pageSize = context.state.ordersList.listConfig.pageSize;
     searchCriteria.sort = context.state.ordersList.listConfig.filters.sort;
+    searchCriteria.startDate = context.state.ordersList.listConfig.filters.startDate;
```

Comment:
There should be a more compact form for copying certain fields from one object to another.
See example:
https://github.com/VirtoCommerce/vc-odt-mpa-theme/blob/dev/client-app/src/pages/account/views/AccountInfo.vue#L63